mantidimaging/gui/windows/spectrum_viewer/view.py

```diff
@@ -334,3 +384,40 @@ def set_binning_visibility(self) -> None:
         self.bin_size_spinBox.setHidden(hide_binning)
         self.bin_step_label.setHidden(hide_binning)
         self.bin_step_spinBox.setHidden(hide_binning)
+
+    def set_roi_properties(self) -> None:
+        current_roi = self.presenter.model.get_roi(self.current_roi)
+        self.roiPropertiesGroupBox.setTitle(self.roiPropertiesGroupBoxTitle + self.current_roi)
+        roi_iter_order = ["Left", "Top", "Right", "Bottom"]
+        row = 0
+        for pos in current_roi.__iter__():
+            with QSignalBlocker(self.roiPropertiesSpinBoxes[roi_iter_order[row]]):
+                self.roiPropertiesSpinBoxes[roi_iter_order[row]].setValue(pos)
+            row = row + 1
+        self.set_roi_spinbox_ranges()
+        self.presenter.redraw_spectrum(self.current_roi)
+        roi_width = abs(self.roiPropertiesSpinBoxes["Right"].value() - self.roiPropertiesSpinBoxes["Left"].value())
+        self.roiPropertiesLabels["Width"].setText(str(roi_width))
+        roi_height = abs(self.roiPropertiesSpinBoxes["Top"].value() - self.roiPropertiesSpinBoxes["Bottom"].value())
```

Comment:
NITPICK: If not too hard and straightforward to do, could we place the width and height at the bottom of the image view and above the spectrum plot inline with the "_x_", "_y_" and "_value_", values (see red text in below image). Sam can properly help you with this if you struggle as I believe he did this in the main window. 

![Screenshot 2024-02-12 at 08 38 58](https://github.com/mantidproject/mantidimaging/assets/32413847/7f3a9e3c-f842-43eb-8800-02046e74ff00)

Reply:
Did you find out how to do this - maybe from Sam?

Reply:
Having been looking at the message setting code recently, I'd see leave it in the side panel for now.

At some we should add a way to make it easier for a window to customise what is shown in the image view message.

Reply:
Yes I think that its something that can be looked at after release as a small change (or potentially big change if it leads to a rabbit hole)

Reply:
See: #2052